book/src/pong-tutorial/pong-tutorial-01.md

```diff
@@ -208,159 +197,54 @@ Then we call `.run()` on `game` which starts the game loop. The game will
-        let color = graph_builder.create_image(
-            window_kind,
-            1,
-            surface_format,
-            // clear screen to black
-            Some(ClearValue::Color([0.0, 0.0, 0.0, 1.0].into())),
-        );
-
-        let depth = graph_builder.create_image(
-            window_kind,
-            1,
-            Format::D32Sfloat,
-            Some(ClearValue::DepthStencil(ClearDepthStencil(1.0, 0))),
-        );
-
-        // Create our single `Subpass`, which is the DrawFlat2D pass.
-        // We pass the subpass builder a description of our pass for construction
-        let pass = graph_builder.add_node(
-            SubpassBuilder::new()
-                .with_group(DrawFlat2DDesc::new().builder())
-                .with_color(color)
-                .with_depth_stencil(depth)
-                .into_pass(),
-        );
-
-        // Finally, add the pass to the graph
-        let _present = graph_builder
-            .add_node(PresentNode::builder(factory, surface, color).with_dependency(pass));
-
-        graph_builder
-    }
-}
+# use amethyst::{prelude::*};
+# fn main() -> Result<(), amethyst::Error>{
+let game_data = GameDataBuilder::default()
+    .with_bundle(
+        RenderingBundle::<DefaultBackend>::new()
+            // The RenderToWindow plugin provides all the scaffolding for opening a window and drawing on it
+            .with_plugin(
+                RenderToWindow::from_config_path(display_config_path)
+                    .with_clear([0.0, 0.0, 0.0, 1.0]),
+            )
+            // RenderFlat2D plugin is used to render entities with `SpriteRender` component.
+            .with_plugin(RenderFlat2D::default()),
+    )?;
+# Ok(()) }
 ```
 
-Here we are creating an `ExampleGraph` struct and implementing a `GraphCreator` trait for it.
-This trait is responsible for setting up all the details of our rendering pipeline.
-
-> **Note:** This setup code is directly using `rendy` crate to define the rendering.
-> You can read about its concepts in the [rendy graph docs][graph].
-
-The important thing to note is that this renders a black background.
-It is also ready to draw 2D sprites for us, which we will use in the next chapter.
+Here we are adding a `RenderingBundle`. Bundles are essentially sets of systems
+preconfigured to work together, so you don't have to write them all down one by one.
 
-If you want to use a different background color, you can tweak the RGBA
-values inside `ClearValue::Color`. Values range from `0.0` to `1.0`,
-so to get that cool green color you can try `[0.00196, 0.23726, 0.21765, 1.0]`.
-
-Now let's pack everything up and run it back in the `main()` function. We have to
-expand the existing `GameDataBuilder` with `RenderingSystem` that uses our graph:
+> **Note:** We will cover systems and bundles in more details later, for now, think of the
+> bundle as a group of functionality that together provides a certain feature to the engine.
```

Comment:
`We will cover systems and bundles in more detail later. For now, think of a bundle as a collection of systems that, in combination, will provide a certain feature to the engine.`